### ISSUE TITLE
Remove character limit from description (backend)

### ISSUE THREAD

**Issue by leonieherma1** [hypha-dao/dho-web-client#1554]
Summary:
As a DAO user I would like to write as much copy as I want in the description

AC:

- [ ] remove description max character limit

<img width="624" alt="image" src="https://user-images.githubusercontent.com/75991832/187916623-5a00abf5-8c9e-4ea6-a2a5-1aa3480dca52.png">

**Comment by leonieherma1** [comment]
> @leonieherma1 Does this mean no limit?

yes @Gerard097  a lot of people copy paste from a google docs / word docs and it pastes the characters weirdly and always only copies 10 sentences and then shows the error

**Comment by leonieherma1** [comment]
> Alright, do you agree if we just double the limit? 5k characters should be more than enough.

sure, let's try it. but there is also something odd going on with copy pasting it into it 

maybe @pugakn can check that?

**Comment by arsenijesavic** [comment]
> > > Alright, do you agree if we just double the limit? 5k characters should be more than enough.
> > 
> > 
> > sure, let's try it. but there is also something odd going on with copy pasting it into it
> > maybe @pugakn can check that?
> 
> Yes, he or @arsenijesavic could give it a look and find why that isn't working as expected

I know what it is, we changed that field a lot and removed the onPaste parser that formats things properly.
Can you add that as the story @leonieherma1 ?

**Comment by leonieherma1** [comment]
@arsenijesavic done https://github.com/hypha-dao/dho-web-client/issues/1566

@Gerard097 can you still double the amount? just in case.... do you want @Alex-IS to look into this?
